presto-native-execution/presto_cpp/main/types/PrestoToVeloxSplit.cpp

```diff
@@ -114,14 +126,55 @@ velox::exec::Split toVeloxSplit(
               : std::optional<std::string>{*entry.second.value});
     }
 
+    std::unordered_map<std::string, std::string> customSplitInfo;
+    customSplitInfo["table_format"] = "hive-iceberg";
+
+    std::vector<velox::connector::hive::iceberg::IcebergDeleteFile> deletes;
+    deletes.reserve(icebergSplit->deletes.size());
+    for (const auto& deleteFile : icebergSplit->deletes) {
+      std::vector<int32_t> equalityFieldIds;
+      equalityFieldIds.reserve(deleteFile.equalityFieldIds.size());
+      for (auto field : deleteFile.equalityFieldIds) {
```

Comment:
Can you try to just use the copy constructor std::vector(const std::vector& another)? List is the same as std::vector<T>